.github/workflows/check-standard.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       - name: Upload check results
```

Comment:
Uploading is also now automatic with v2: see [the v2 release notes](https://github.com/r-lib/actions/releases/tag/v2) for all changes (search for "upload" to find that one).

Reply:
Oh nice, I actually have not used this action for some reason... will fix